stock_request_analytic/models/stock_request.py

```diff
@@ -1,30 +1,20 @@
-# Copyright 2017 Eficent Business and IT Consulting Services, S.L.
+# Copyright 2017-2020 ForgeFlow, S.L. (https://www.forgeflow.com)
 # License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
 
-from odoo import _, api, fields, models
-from odoo.exceptions import ValidationError
+from odoo import fields, models
 
 
 class StockRequest(models.Model):
     _inherit = "stock.request"
```

Comment:
I think you need to add `_check_company_auto = True` here for the check to work: https://www.odoo.com/documentation/13.0/howtos/company.html?highlight=multi%20company#multi-company-consistency